components/private_channel/rust/ffi/src/private_channel.cc

```diff
@@ -0,0 +1,81 @@
+/* Copyright 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+#include "private_channel.h"  // NOLINT
```

Comment:
there should be a space below the copyright and also between private_channel.h and iostream